sdk/cognitivelanguage/Azure.AI.Language.Conversations/tests/ConversationAnalysisClientLiveTests.cs

```diff
@@ -23,30 +23,30 @@ public async Task AnalyzeConversation()
         {
             var data = new
             {
-                analysisInput = new
+                AnalysisInput = new
                 {
-                    conversationItem = new
+                    ConversationItem = new
                     {
-                        text = "Send an email to Carol about the tomorrow's demo",
-                        id = "1",
-                        participantId = "1",
+                        Text = "Send an email to Carol about the tomorrow's demo",
+                        Id = "1",
```

Comment:
This is one of several reasons I didn't like this as a default: I see "ID" much more often than "Id". It doesn't even matter what .NET guidelines say. It matters what people do. "ID" is a common pseudo-acronym that we'd like improperly case as "iD".

Given these are my live tests - not samples - I'd rather leave them as-is, or is the fear people will stumble across them?

Reply:
Using a property called "ID" with the CamelCase option would write a JSON property called "id", which I think is what's expected.